code/index.html

```diff
@@ -2,21 +2,155 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>HelsinkiSaunas</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script type="module" src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"></script>
+    <script nomodule src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.js"></script>
   </head>
+  
   <body>
     <header>
-      <!-- Some kind of logo and navbar -->
+      <div class="logo-mobile"><span>Fake logo</span></div>
+      <h1 class='site-name'>HelsinkiSaunas</h1>
+      <nav class="navbar-tablet">
+        <a href="#">Home</a>
+        <a href="#">About</a>
+        <div class="logo-tablet"><span>Fake logo</span></div> 
+        <a href="#">Saunas</a>
+        <a href="#">Info</a>
+      </nav>
+      <nav class="navbar-desktop">
+        <a class="links-style-desktop"href="#">Home</a>
+        <a class="links-style-desktop"href="#">About</a> 
+        <a class="links-style-desktop"href="#">Saunas</a>
+        <a class="links-style-desktop"href="#">Info</a>
+      </nav>
+      <ion-icon class="menu-icon" name="menu-outline"></ion-icon>
     </header>
 
+    
     <section class="big-news">
-      <!-- The big news content -->
+     <div class="big-news-image-container">
+        <a style="image-link-style"href="https://allasseapool.fi/?lang=en"target="_blank">
+        <div class="polaroid" polaroid-caption="#AllasSeaPool">
+          <img class="big-news-image" src="https://allasseapool.fi/wp-content/uploads/2021/06/allas-2016-opening_party-eetu_ahanen-045-1-1-scaled.jpg">
+        </div>
+        </a>
+      </div>
+      
+      <div class="big-news-text-container">
+        <div class="big-news-text-tablets-desktops">
+        <h1 class="big-news-title">Sea spa and an oasis for wellbeing nestled in the heart of Helsinki</h1>
```

Comment:
Using more than one h1 in your page is totally fine, but it's not best practice, especially not effective for screen reader. As h1 is to describe the the web page (similar to <title> tag). You may find this article helpful: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements